uasc/message_test.go

```diff
@@ -18,7 +18,14 @@ func TestMessage(t *testing.T) {
 		{
 			Name: "OPN",
 			Struct: func() interface{} {
-				m := NewMessage(
+				m := (&SecureChannel{
```

Comment:
This is too much magic for my taste. Can you decouple that a bit?

Reply:
extracted SecureChannel into a variable